embryo/segment_2D_0.py

```diff
@@ -38,23 +31,23 @@
 # Contouring does not reveal as many details
 
 # %%
-# Try various global thresholding techniques
-fig, ax = try_all_threshold(frame, figsize=(10, 10), verbose=False)
+# Try all global thresholding techniques
+fig, ax = ski.filters.try_all_threshold(frame, figsize=(10, 10), verbose=False)
 plt.show()
 
 # %% [markdown]
 # Isodata and Otsu seem to give the best results.
```

Comment:
Maybe we don't know what kind of 'results' are expected at this stage... Could we be more descriptive here? What about: "Isodata and Otsu seem to be the best candidates for filtering the objects of interest from the background?"